workbook.go

```diff
@@ -130,6 +130,89 @@ func (f *File) UnprotectWorkbook(password ...string) error {
 	return err
 }
 
+// unselectSheetTab deselects the sheet tab at the specified sheet index
+// if it is currently active.
+func (f *File) unselectSheetTab(sheetIdx int, wb *xlsxWorkbook) {
+	activeSheetName := f.GetSheetName(f.GetActiveSheetIndex())
+	if activeSheetName == wb.Sheets.Sheet[sheetIdx].Name {
+		ws, err := f.workSheetReader(wb.Sheets.Sheet[sheetIdx].Name)
+		if err != nil {
+			return
+		}
+		ws.SheetViews.SheetView[0].TabSelected = false
+	}
+}
+
+// SwapSheets swaps the position of two sheets in the workbook
+// based on their index positions. If the indices of the sheets
+// are the same, the function does nothing. Note that the index
+// must be greater than or equal to 0 and less than SheetCount.
+// For example:
+//
+//	err := f.SwapSheets(1, 3)
+//	if err != nil {
+//	  // handle the error
+//	}
+func (f *File) SwapSheets(sheet1Idx, sheet2Idx int) error {
+	if sheet1Idx == sheet2Idx {
+		return nil
+	}
+
+	wb, err := f.workbookReader()
+	if err != nil {
+		return err
+	}
+	if sheet1Idx >= f.SheetCount || sheet2Idx >= f.SheetCount || sheet1Idx < 0 || sheet2Idx < 0 {
+		return ErrSheetIdx
+	}
+
+	// Unselect the tab of a sheet by index
```

Comment:
Move sheet may break grouped sheets, so we just ungroup all sheets instead of unselecting each sheet, so the `unselectSheetTab` function no longer used.